src/nvim/ex_docmd.c

```diff
@@ -9860,6 +9860,12 @@ bool cmd_can_preview(char_u *cmd)
     return false;
```

Comment:
The doc could also be changed to:

    /// @param[in] cmd Commandline to check. May start with a range or modifier.

Reply:
Doc updated and tests added.